tests/integration/test_api_gateway.py

```diff
@@ -127,3 +148,38 @@ def forward_request(self, **kwargs):
 
     # clean up
     proxy.stop()
+
+
+def test_api_gateway_lambda_proxy_integration():
+    test_port = 12123
+    backend_url = 'http://localhost:%s%s' % (test_port, API_PATH_LAMBDA_PROXY_BACKEND)
+
+    # create target HTTP backend
+    class TestListener(ProxyListener):
+
+        def forward_request(self, **kwargs):
+            response = Response()
+            response.status_code = 200
+            response._content = json.dumps(kwargs['data']) if kwargs['data'] else '{}'
+            return response
+
+    proxy = GenericProxy(test_port, update_listener=TestListener())
+    proxy.start()
+
+    # create API Gateway and connect it to the Lambda proxy backend
+    result = connect_api_gateway_to_http_with_lambda_proxy('test_gateway2', backend_url,
```

Comment:
Instead of passing in `backend_url`, we probably need to pass in the ARN of the Lambda function here: http://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arn-syntax-lambda . See utility methods to create ARNs here: https://github.com/localstack/localstack/blob/master/localstack/utils/aws/aws_stack.py#L247